MedvedevKS/lab_4/lab_4.cpp

```diff
@@ -0,0 +1,105 @@
+// lab_4.cpp: ���������� ����� ����� ��� ����������� ����������.
+//
+
+#include "stdafx.h"
+#include <conio.h>
+#include <iostream>
+using namespace std;
+
+double intPower(double number, int n)
+{
+	double result = 1;
+	for (int i = 0; i < abs(n); i++)
+	{
+		result *= number;
+	}
+	if (n < 0)
+	{
+		result = 1 / result;
+	}
+	return result;
+}
+
+int factorial(int number)
+{
+	if (number < 0)
+	{
+		return 0;
+	}
+	if (number > 12)
+	{
+		return -1;
+	}
+	int result = 1;
+	for (int i = 2; i <= number; i++)
+	{
+		result *= i;
+	}
+	return result;
+}
+
+double SquareRoot(double number)
+{
+	double result = 0;
+	for (int i = 0; i < 10000; i++)
+	{
+		result = 1 + (number - 1) / (result + 1);
+	}
+	return result;
+}
+
+int PrimeCheck(int number)
+{
+	if (number <= 1)
```

Comment:
One is considered to be prime, so you can easely make your function to return bool

Reply:
is 1 considered to be prime? It's not prime and not compound if I'm not mistaken

Reply:
yep, my bad